src/main/java/seedu/address/model/util/SampleDataUtil.java

```
@@ -0,0 +1,53 @@
+                    new Address("Blk 30 Geylang Street 29, #06-40"),
+                    new UniqueTagList("friends")),
+                new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@gmail.com"),
+                    new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
+                    new UniqueTagList("colleagues", "friends")),
+                new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@yahoo.com"),
+                    new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
+                    new UniqueTagList("neighbours")),
+                new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@google.com"),
+                    new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
+                    new UniqueTagList("family")),
+                new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@outlook.com"),
+                    new Address("Blk 47 Tampines Street 20, #17-35"),
+                    new UniqueTagList("classmates")),
+                new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@gmail.com"),
+                    new Address("Blk 45 Aljunied Street 85, #11-31"),
+                    new UniqueTagList("colleagues"))
+            };
+        } catch (IllegalValueException e) {
+            throw new AssertionError("sample data cannot be invalid", e);
+        }
+    }
+
+    public static ReadOnlyAddressBook getSampleAddressBook() {
+        try {
+            AddressBook sampleAB = new AddressBook();
+            for (Person samplePerson : getSamplePersons()) {
+                sampleAB.addPerson(samplePerson);
+            }
+            return sampleAB;
+        } catch (DuplicatePersonException e) {
+            throw new AssertionError("sample data cannot contain duplicate persons", e);
```

@edmundmok 
> So in general, if we caught an impossible exception like in this scenario, it would be more appropriate to handle it with an assertionError instead of just an assert?

Yes. `AssertionError` will always be thrown, while `assert` will only work when it is enabled. So `assert` should only be used for "extra checks". In this case, we are already performing the "duplicate person" check, so there is no point using `assert` since we are not gaining any performance or anything. In fact, it is worse as if `assert` is disabled, then we are essentially ignoring this exception which is clearly signaling that something is wrong.